### ISSUE TITLE
eslint no-undef when using placeholder args

### ISSUE THREAD

**Issue by plexigras** [wcjohnson/lightscript#68]
```js
[1,2,3].map(-> _*2)
```
```
1:16  error  '_' is not defined  no-undef
```

**Comment by wcjohnson** [comment]
Test in a clean room environment with eslint semver 4.8 (not 4.9 or newer) and let me know if this persists. If not, then it's related to Babel 7 stuff.

**Comment by wcjohnson** [comment]
See #67

**Comment by wcjohnson** [comment]
Fixed with the new 4.0 linter architecture.
